### PR TITLE
Ug bug fixes

### PR DESCRIPTION
**Pull request by notle1706**

### CODE REVIEW

**Review by jicsontoh — APPROVED** [review]
LGTM, good catch!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ Format: `add n/NAME a/ADDRESS p/PHONE e/EMAIL [t/TAG]...`
 * No restrictions on the phone input field and email input field, but a warning will be given if it deviates from the standard convention.
     * This facilitates more freedom to input phone numbers like `+606 89987755 (HOME)` and emails like `alice@company.com (WORK)`
 
-> <div markdown="span" class="alert alert-danger">❗ :warning: **You cannot add a client with a name that already exists in JeeqTracker**: Names are considered duplicates even if they differ by case sensitivity or white spaces!
+> <div markdown="span" class="alert alert-danger">❗ :warning: **You cannot add a client with a name that already exists in JeeqTracker**: Names are considered duplicates even if they differ by case sensitivity or whitespaces!
```
Comment (Jo3LW) [inline docs/UserGuide.md:128]: Great catch!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -175,8 +175,7 @@ Example:
 
 * `clear` clears all Client's entries together with the Remarks and Transactions.
 
-> <div markdown="span" class="alert alert-danger">❗ :warning: Information cleared by clear command cannot be retrie
- </div>
+> <div markdown="span" class="alert alert-danger">❗ :warning: Information cleared by clear command cannot be retrieved
```
Comment (Jo3LW) [inline docs/UserGuide.md]: Good find! Would have been a avoidable bug :)

Comment (jicsontoh) [reply docs/UserGuide.md]: I agree

File: docs/UserGuide.md (hunk 1 above)
Comment (notle1706) [reply docs/UserGuide.md:128]: Thank you!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -175,8 +175,7 @@ Example:
 
 * `clear` clears all Client's entries together with the Remarks and Transactions.
 
-> <div markdown="span" class="alert alert-danger">❗ :warning: Information cleared by clear command cannot be retrie
- </div>
+> <div markdown="span" class="alert alert-danger">❗ :warning: Information cleared by clear command cannot be retrieved!
```
Comment (Jo3LW) [inline docs/UserGuide.md:178]: Good find! This bug would have been an avoidable bug :)

**Review by Jo3LW — APPROVED** [review]
LGTM!

File: docs/UserGuide.md (hunk 2 above)
Comment (notle1706) [reply docs/UserGuide.md]: I added an exclamation mark in the latest commit!

Comment (jicsontoh) [reply docs/UserGuide.md]: nice! makes the alert look better

File: docs/UserGuide.md (hunk 1 above)
Comment (quannam0124) [reply docs/UserGuide.md:128]: Very insightful warning!

**Pull request merged**